include/swift/AST/DiagnosticsSIL.def

```diff
@@ -352,6 +352,10 @@ ERROR(global_must_be_compile_time_const,none,
       "global variable must be a compile-time constant", ())
 ERROR(non_final_generic_class_function,none,
       "classes cannot have non-final generic fuctions in embedded Swift", ())
+ERROR(embedded_swift_existential_type,none,
+      "Existential type %0 is unavailable in embedded Swift", (Type))
```

Comment:
nitpick: I think we don't use "existential" as user facing term. There are a few other error messages which contain "existential", but I think it's better to call it "protocol-type" or "protocol as type", like it's done for other sema error messages.

Reply:
`value of protocol type %0` is what we use in Sema.

Another nit: the message shouldn't be capitalized.

Reply:
How about this wording?
```
 ERROR(embedded_swift_existential_type,none,
       "using values of protocol type %0 is not allowed in embedded Swift", (Type))
 ERROR(embedded_swift_existential,none,
       "using values of protocol types is not allowed in embedded Swift", ())
 ```